skyfield/nutationlib.py

```diff
@@ -29,6 +29,32 @@ def compute_nutation(t):
                    cpsi * cobm * sobt - sobm * cobt,
                    cpsi * sobm * sobt + cobm * cobt)))
 
+def compute_ECLIP(t):
+    """Generate the ecliptic rotations for Time `t`.
+
+    If the Julian date is scalar, a simple ``(3, 3)`` matrix is
+    returned; if the date is an array of length ``n``, then an array of
+    matrices is returned with dimensions ``(3, 3, n)``.
+
+    """
+    oblm, oblt, eqeq, psi, eps = earth_tilt(t)
+    e = oblt*DEG2RAD
+    coe = cos(e)
+    sie = sin(e)
+    if not t.shape:
```

Comment:
Can you use the routine `rot_x()` in the `functions` module for this, by any chance? I tried to design that routine so that it could stand in for this entire `if` statement, and let code authors avoid having to re-implement basic rotation matrices.

Reply:
Ok, it makes sense.

Reply:
Unfortunately I ended up having problems with this part. 
If I am doing calculations with an array of times, then the angle for obliquity of the ecliptic from e = oblt * DEG2RAD is array right? So when calling rot_x(e) - I end up with an error ValueError: setting an array element with a sequence. (I am clueless as to how that everything is an array numpy magic works)